rand_distr/Cargo.toml

```diff
@@ -45,3 +45,5 @@ rand = { path = "..", version = "=0.9.0-alpha.1", features = ["small_rng"] }
 average = { version = "0.15", features = [ "std" ] }
 # Special functions for testing distributions
 special = "0.10.3"
+# For analytical cdf calculations
+statrs = { version = "0.17.1", git="https://github.com/statrs-dev/statrs", default-features = false}
```

Comment:
You should [specify](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#choice-of-commit) a `rev` (or maybe `tag` or *possibly* `branch` if that's stable); if left like this it's quite likely that builds will fail in the future.

We also can't make a release with this in place, hence I'd prefer not to merge for now.

Reply:
I have added an implementation of the cdf. We can depend on statrs later.

(Maybe we want to provide the cdf also for the user eventually)

Reply:
> Maybe we want to provide the cdf also for the user eventually

That's beyond the scope I want for this project (also, `rand_distr` should IMO focus on sampling efficiently, not anything else).